### PR TITLE
Bug/gsye 61

### PR DESCRIPTION
**Pull request by spyrostz**
Ported the is_time_slot_in_simulation_duration method to gsy-framework in order to be reused by the framework itself when generating market slots for the simulation duration.

### CODE REVIEW

File: gsy_framework/utils.py [hunk 1]
```diff
@@ -544,3 +544,21 @@ def sort_list_of_dicts_by_attribute(input_list: List[Dict], attribute: str, reve
...
+
+def is_time_slot_in_simulation_duration(
+        time_slot: DateTime, config: "GlobalConfig" = None) -> bool:
+    """
+    Check whether a specific timeslot is inside the range of the simulation duration.
+    :param time_slot: Timeslot that is checked if it is in the simulation duration
+    :param config: Optional configuration settings object that is used for the start / end date
+                   parameters. In case it is not provided, GlobalConfig object is used instead
+    :return: True if the timeslot is in the simulation duration, false otherwise
+    """
```
Comment (hannesdiedrich) [inline gsy_framework/utils.py:557]: NIT: Please use google docstring style. Thanks!

File: gsy_framework/utils.py [hunk 2]
```diff
@@ -544,3 +544,21 @@ def sort_list_of_dicts_by_attribute(input_list: List[Dict], attribute: str, reve
 def convert_datetime_to_ui_str_format(data_time):
     """Convert a datetime object into the format required by the UI."""
     return instance(data_time).format(DATE_TIME_UI_FORMAT)
+
+
+def is_time_slot_in_simulation_duration(
```
Comment (hannesdiedrich) [inline gsy_framework/utils.py:549]: Please also create a follow up PR for the gsy-e where you use this function. Thanks a lot!

**Review by hannesdiedrich — APPROVED** [review]
Two comments, LGTM though

File: gsy_framework/utils.py (hunk 1 above)
Comment (spyrostz) [reply gsy_framework/utils.py:557]: Thanks, fixing

**Review by fievelk — APPROVED** [review]

**Pull request merged**